less/variables.less

```diff
@@ -1,8 +1,8 @@
 @big: ~"only screen and (min-width: 1400px) and (max-width: 3000px)";
 @desktop: ~"only screen and (min-width: 1100px) and (max-width: 1399px)";
-@tablet: ~"only screen and (min-width: 620px) and (max-width: 1099px)";
-@phone: ~"only screen and (max-width: 619px)";
-@phone-max: 620px;
+@tablet: ~"only screen and (min-width: 701px) and (max-width: 1099px)";
+@phone: ~"only screen and (max-width: 700px)";
+@phone-max: 700px;
```

Comment:
I don't think it will ba a value like that, but rather a standard one, like 768px, 640px, etc. Don't you think?